src/components/product/ProductGrid.tsx

```diff
@@ -4,7 +4,6 @@ import React, { useEffect, useState } from 'react';
 
 import { CanceledError } from '@/lib/services/apiClient';
 import { Error } from '@/components/apiResponseState/Error';
-import { Loader } from '@/components/apiResponseState/Loader';
```

Comment:
loader is no longer needed here, but the component still needs to keep information about its laoding state (I think)